1/contrib/jenkins/kube-slave-common.sh

```diff
@@ -68,12 +68,21 @@ function get_is_names() {
   $oc_cmd get is -n "${PROJECT_NAME}" -l role=${SLAVE_LABEL} -o template --template "{{range .items}}{{.metadata.name}} {{end}}"
 }
 
+# get_istag_names returns a list of imagestreamtags that match the SLAVE_LABEL
+# role
+function get_istag_names() {
+  [ -z "$oc_cmd" ] && return
```

Comment:
This is clearly not your invention but it's unclear why it needs to be in every single function, seems more like a clear precondition for any part of this script working